index.html

```diff
@@ -2558,6 +2558,90 @@ <h3>serviceEndpoint</h3>
+                        z+3uTRUYyuRiLdvvc7Myjq7CrNkiqr5yedMXwM9Bi3GVUIZ3L4MKgyCf7XZ0TMoX
+                        i4FVWbi4O1caKM0foglpRzGQM0IPgwQ/JR5u+rWaNx5XdowraWqU0LziznibvuFI
+                        HTpcQVmO0zNDAgMBAAEwDQYJKoZIhvcNAQELBQADggEBAAJPFM32NBdAsGnuChQ9
+                        1A66HvVwQeC/r2wJFVj3jg7h7lTYjmaHKoRP3J5UGmHo8fTbIxyJPKoXZ3qoj31Y
+                        uBFKxNaNjSrPTOUv/4injvqjkdCvQdy0iuQaNtQJSkse4t/fnQ+DMC+E3cNii8Yj
+                        0Z+T3oJL1PCg01N6uL8KtooEmnUGzIzuZ2naokgwaiEOU2CCnJOCKYxz22cxqfNm
+                        sGy1keqt6/bE64UnJRgETdqHzZTO0013ZtvDiwjGo3CZg4FBi/O722XnxbzuFIPZ
+                        WWOOb9VaH+AE4lNei6i/CDY/WcgpP0GDIj3z6lCeLtiSagiQQPYxIz2sBxC1OKVU
+                        7CE=" ],
+                        "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..ktzmVCTovhKWKxnMwDtMnmpUAUF1n7sqbdNmOV0TkxCTb2eHATO6iP7wqjyl_CMehHKnqgW6t9SW4q9TdbCxAg",
+                        "proofPurpose": "assertionMethod",
+                        "verificationMethod": "did:key:z6MkkUNcUWxAbxamK2Spp92mKWoXddS3tTqBKe8VTGExzX2Y#z6MkkUNcUWxAbxamK2Spp92mKWoXddS3tTqBKe8VTGExzX2Y"
+                      }
+                    }
+                  </pre>
+      </section>
+
+      <section id="x509CertificateFingerprint" about="https://w3id.org/security#x509CertificateFingerprint"
+      typeof="rdf:Property">
+        <h3>x509CertificateFingerprint</h3>
+        <p>
+          The x509CertificateFingerprint property is used to associate an <a href="https://tools.ietf.org/html/rfc5280">X.509 Certificate</a> with a proof via its fingerprint.
+          The value is a <a href="https://tools.ietf.org/id/draft-multiformats-multihash-00.html#rfc.appendix.B.1">multihash</a> encoded then <a href="https://tools.ietf.org/id/draft-multiformats-multibase-00.html">multibase</a> 
+          encoded value using the <a href="https://tools.ietf.org/id/draft-multiformats-multibase-00.html#rfc.appendix.D.1">base64pad</a> varient. It is RECOMMENDED that the fingerprint value be the SHA-256
+          hash of the X.509 Certificate.
+                  </p>
+                  <dl>
+                    <dt>Status</dt>
+                    <dd property="vs:term_status">unstable</dd>
+                    <dt>Range</dt>
+                    <dd>Signature</dd>
+                  </dl>
+                  <p>
+          The following example demonstrates how a proof is expressed using
+          a JSON-LD proof with a link to an associated <code>x509CertificateFingerprint</code>:
+                  </p>
+                  <pre class="example prettyprint language-jsonld">
+                    {
+                      ...
+                      "proof": {
+                        "type": "Ed25519Signature2018",
+                        "created": "2020-04-03T18:03:33Z",
+                        "x509CertificateFingerprint": "egW8ixMpQ2W7SG46Y+ZzPkCUMjxmq7WIuMego9CZdhU=",
```

Comment:
Note that I think this needs to start with `M` for base64pad.

Reply:
https://tools.ietf.org/html/draft-multiformats-multibase-03

I might recommend base64url instead... which you be `u`.

Reply:
> I might recommend base64url instead... which you be u.

We had suggested base64pad because that's what the default is for PEM encoding and for the `x509CertificateChain` field. I don't have a strong opinion either way, I expect folks that do base64pad also have base64url libraries available to them... and arbitrary base-encoding is fairly easy to implement.

The only rationale that I can think of for base64url by default is that we think it's going to end up in a URL somewhere. I don't expect it to, and if it does, I expect folks will URI-encode base64pad values?

The rationale for base64pad is because "that's what PEM-encoded x509 values are encoded as".

Leaning ever so slightly towards base64pad by default.... but again, no real strong feelings.

Reply:
If the default string encoding is usable by just dropping the `M` that seems like a good enough justification, agree regarding URLs

Reply:
Updated now thanks

Reply:
I think continuing with base64pad for now to align better with PEM is the best approach, JOSE uses a mix of base64url and pad for x509 properties, so there is precedent there.